article/app/views/fragments/recipeArticleBody.scala.html

```diff
@@ -25,10 +25,10 @@
 
     <div class="recipes__main">
         <div class="recipe__images-wrapper">
-            <div class="recipe__label">
-                <h2>food &amp; drink</h2>
-            </div>
-            <div class="recipe__image__wrapper recipe__image__wrapper--is-displayed">
+            <a href="https://www.theguardian.com/tone/recipes?CMP=recipe-article">
```

Comment:
@SiAdcock This is something which doesn't exist in the data model anywhere so i dont think we can do it dynamically. Which part is troubling for you?